### PR TITLE
fix:clean up addOrdinaSuffix and add error handling

### LINKED ISSUE #42: Error handling issue
@ali-master

I've read the source of the project, and I realize the project missing `Error handling`.
what your think about that.

I'm looking forward to hearing from you.

### PR DESCRIPTION
**Pull request by masoudDaliriyan**
Hi, All 
I think we should take care of error messages and raise more readable errors.

### CODE REVIEW

**Comment by ali-master** [comment]
Hi @masoudDaliriyan jan,

Prettier CI has failed in [CI Action](https://github.com/persian-tools/persian-tools/pull/48/checks?check_run_id=2043431448)

**Comment by masoudDaliriyan** [comment]
What can i do?

**Comment by ali-master** [comment]
Please be aware of conventional commit messaging.
```diff
- fix:clean up addOrdinaSuffix and add error handling
+ feat(addOrdinalSuffix): add error handling and minor improvement
```

**Comment by masoudDaliriyan** [comment]
The error is not clear

**Comment by masoudDaliriyan** [comment]
> Please be aware of conventional commit messaging.
> ```diff
> - fix:clean up addOrdinaSuffix and add error handling
> + feat(addOrdinalSuffix): add error handling and minor improvement
> ```

Ok, Thanks for advice

**Comment by ali-master** [comment]
> What can i do?

Fix the reported Prettier's error on your file and push it back.

File: src/modules/addOrdinalSuffix/addOrdinalSuffix.ts [hunk 1]
```diff
@@ -4,16 +4,20 @@
...
-		number += " اُم";
-	} else if (number.endsWith("سه")) {
-		number = number.slice(0, -2) + "سوم";
-	} else number += "م";
+    if (number.endsWith("ی")) {
+        return number + " اُم"
+    }
 
-	return number;
+    if (number.endsWith("سه")) {
```
Comment (ali-master) [inline src/modules/addOrdinalSuffix/addOrdinalSuffix.ts]: You should write multiple test-cases to cover this expression and all lines into the block.

**Comment by ali-master** [comment]
Merged. Thanks

**Pull request merged**